### PR TITLE
parser: add syntax support for show charset

### PR DESCRIPTION
**Pull request by holys**
this fix https://github.com/pingcap/docs-cn/issues/187

@shenli  @tiancaiamao  @zimulala  @hanfei1991   PTAL

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -5158,6 +5158,10 @@ ShowTargetFilterable:
 	{
 		$$ = &ast.ShowStmt{Tp: ast.ShowCharset}
 	}
+|	"CHARSET"
```
Comment (shenli) [inline parser/parser.y]: You can define a new rule such as:
CharsetSym:
"CHARSET" | "CHARACTER" "SET"

File: parser/parser.y [hunk 2]
```diff
@@ -5322,6 +5322,9 @@ ShowTableAliasOpt:
 		$$ = $2.(*ast.TableName)
 	}
 
+CharsetSym:
```
Comment (tiancaiamao) [inline parser/parser.y]: You should declare it in 
```
%type   <item>
```

**Comment by tiancaiamao** [comment]
I find there is already a `CharsetKw` in parser.y @holys

**Comment by holys** [comment]
@tiancaiamao  So keep `CharsetKw` instead of `CharsetSym ` ?

**Comment by tiancaiamao** [comment]
Yes, there is no need to define a new one as we already have it.

**Comment by hanfei1991** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**